lib/src/configuration.dart

```diff
@@ -84,45 +65,34 @@ class Configuration {
     return "";
   }
 
-  @override
-  bool operator ==(Object other) {
-    if (identical(this, other)) return true;
-    if (other is! Configuration) return false;
-    return realmCore.configurationEquals(this, other);
-  }
-
   /// The schema version used to open the [Realm]
   ///
   /// If omitted the default value of `0` is used to open the [Realm]
   /// It is required to specify a schema version when initializing an existing
   /// Realm with a schema that contains objects that differ from their previous
   /// specification. If the schema was updated and the schemaVersion was not,
   /// an [RealmException] will be thrown.
-  int get schemaVersion => realmCore.getSchemaVersion(this);
-  set schemaVersion(int value) => realmCore.setSchemaVersion(this, value);
+  int schemaVersion;
```

Comment:
```suggestion
  final int schemaVersion;
```
and initialize in ctor .. and use the @immutable annotation from `meta` package.